### PR TITLE
feat(server): correctly parse router log messages from stream

### PR DESCRIPTION
**Pull request by jchauncey**

### CODE REVIEW

File: influx/queries_per_second.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+// This code isnt being used right now since we can perform the calculations in grafana. However, we are keeping this around just in case we need it in a future release.
+
+//QueriesPerSecond finds the qps value for the aggregate and individual endpoints
+func QueriesPerSecond(influxClient client.Client) error {
+	err := writeAggregateQueriesPerSecond(influxClient)
+	if err != nil {
+		return err
+	}
+	err = writePerAppQueriesPerSecond(influxClient)
+	if err != nil {
```
Comment (arschles) [inline influx/queries_per_second.go:26]: general note, not blocking for this PR - you can save a line by writing `if err := writePerAppQueriesPerSecond(influxClient); err != nil {`

File: syslogish/server.go [hunk 2]
```diff
@@ -1,27 +1,31 @@
...
+	"github.com/deis/stdout-metrics/influx"
+	"github.com/deis/stdout-metrics/util"
+	"github.com/influxdata/influxdb/client/v2"
 )
 
 const (
 	bindHost  = "0.0.0.0"
 	bindPort  = 1514
-	queueSize = 500
+	queueSize = 50000
```
Comment (arschles) [inline syslogish/server.go:17]: not blocking for this PR - worth making this and/or the above params configurable?

Comment (jchauncey) [inline syslogish/server.go:17]: yeah i want to come back to this in rc1 and clean this up some more.

File: util/util.go [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+func parseInt(s string) int {
+	v, _ := strconv.Atoi(s)
+	return v
+}
+
+func parseFloat(s string) float64 {
+	v, _ := strconv.ParseFloat(s, 64)
+	return v
+}
```
Comment (arschles) [inline util/util.go:95]: why ignore the errors in the above three functions?

File: syslogish/server.go (hunk 2 above)
Comment (arschles) [inline syslogish/server.go:17]: issue for that cleanup?

**Comment by arschles** [comment]
left a few comments, but my only one blocking is https://github.com/deis/stdout-metrics/pull/3#discussion_r62565658

File: util/util.go (hunk 3 above)
Comment (jchauncey) [inline util/util.go:95]: particularly because i loathe double returns liek this because i cant one line an assignment..

Comment (arschles) [inline util/util.go:95]: so, how are you gonna handle errors parsing times, ints and floats (respectively)?

Comment (jchauncey) [inline util/util.go:95]: updated

File: syslogish/server.go (hunk 2 above)
Comment (jchauncey) [inline syslogish/server.go:17]: https://github.com/deis/stdout-metrics/issues/5

**Comment by jchauncey** [comment]
@arschles updated code to just remove those fucntions and make the checks while parsing

**Pull request merged**